docs/topics/broad-crawls.rst

```diff
@@ -39,18 +39,16 @@ you need to keep in mind when using Scrapy for doing broad crawls, along with
 concrete suggestions of Scrapy settings to tune in order to achieve an
 efficient broad crawl.
 
+
 .. _broad-crawls-scheduler-priority-queue:
 
 Use the right :setting:`SCHEDULER_PRIORITY_QUEUE`
 =================================================
 
-Scrapy’s default scheduler priority queue is ``'scrapy.pqueues.ScrapyPriorityQueue'``.
-It works best during single-domain crawl. It does not work well with crawling
-many different domains in parallel
-
-To apply the recommended priority queue use::
+When crawling multiple domains, setting the :setting:`SCHEDULER_PRIORITY_QUEUE`
+setting to :class:`~scrapy.pqueues.DownloaderAwarePriorityQueue` can
+significantly reduce crawl time.
 
-    SCHEDULER_PRIORITY_QUEUE = 'scrapy.pqueues.DownloaderAwarePriorityQueue'
```

Comment:
I think that's nice to keep an example which users can copy-paste, even if it is redundant.